Project.toml

```diff
@@ -60,7 +60,7 @@ OrderedCollections = "1.1"
 PencilArrays = "0.16, 0.17"
 PencilFFTs = "0.13.5, 0.14"
 Rotations = "1.0"
-SeawaterPolynomials = "0.3"
+SeawaterPolynomials = "0.3.2"
```

Comment:
we need to change the Manifest also.

Reply:
Only changing the Project compat entry won't force the package to update. So doctests would probably fail until you also actually update the package and push the new Manifest...

Reply:
Ah, I see! I have not done this before but I assume you can just edit the `Manifest.toml` entry 

```
[[deps.SeawaterPolynomials]]
git-tree-sha1 = "3e4e6c809e96ddcc0077bdb6944a1abb53fc382b"
uuid = "d496a93d-167e-4197-9f49-d3af4ff8fe40"
version = "0.3.0"
```

where version = "0.3.0" -> version = "0.3.2"?

Reply:
> Only changing the Project compat entry won't force the package to update. So doctests would probably fail until you also actually update the package and push the new Manifest...

Ok, I did think Manifest was usually off limits. Will do as you suggest and update the package, then push.

Reply:
No don't manually edit the manifest! Just update the SeawaterPolynomials locally. This would alter the manifest. ;)

Reply:
It may be that several other dependencies need to be updated.

Reply:
> No don't manually edit the manifest! Just update the SeawaterPolynomials locally. This would alter the manifest. ;)

Don't worry, the only thing in know about the Manifest is not touch the Manifest so would only have done this if I was explicitly told to do so.